packages/frontend/src/components/AboutUsModal.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+    Button,
+    Modal,
+    ModalBody,
+    ModalFooter,
+    ModalHeader,
+    ModalTitle,
+} from "@alphaday/ui-kit";
+
+interface IProps {
+    showModal: boolean;
+    onClose?: () => void;
+}
+export const AboutUsModal: React.FC<IProps> = ({ showModal, onClose }) => {
+    return (
+        <Modal showModal={showModal} onClose={onClose}>
+            <ModalHeader className="">
+                <ModalTitle>About Us</ModalTitle>
+            </ModalHeader>
+
+            <ModalBody className="[&_a]:text-secondaryOrange">
+                <h6>Company Overview:</h6>
+                <p>
+                    Registered Name: Alphabox Solutions Pte. Ltd.
+                    <br />
+                    Registration Number: <code>202136261C</code>
+                    <br />
+                    Registered Office: 45 North Canal Road #01-01 Lew Building
+                    Singapore
+                </p>
+
+                <h6>Contact Information:</h6>
+                <p>
+                    Email Address:{" "}
+                    <a href="mailto:hello@alphaday.com">hello@alphaday.com</a>
+                    <br />
+                    X.com:{" "}
+                    <a
+                        href="https://x.com/AlphadayHQ"
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        https://x.com/AlphadayHQ
+                    </a>
+                </p>
+
+                <h6>Legal Information:</h6>
+                <p>
+                    Terms & Conditions:{" "}
+                    <a
+                        href="https://alphaday.com/terms"
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        https://alphaday.com/terms
+                    </a>
+                    <br />
+                    Privacy Policy:{" "}
+                    <a
+                        href="https://alphaday.com/privacy"
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        https://alphaday.com/privacy
+                    </a>
+                </p>
+
+                <h6>Additional Resources:</h6>
+                <p>
+                    FAQs:{" "}
+                    <a
+                        href="https://alphaday.com/"
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        alphaday.com
+                    </a>
+                    <br />
+                    API:{" "}
+                    <a
+                        href="http://api.alphaday.com/"
+                        target="_blank"
+                        rel="noreferrer"
```

Comment:
Please add a `"nofollow"` as well

Reply:
not sure if I get why (?)

Reply:
I see what you mean, it makes sense, but this html shouldn't be rendered to crawlers. Also, I'm waiting for Deniz's response, because I'd prefer to remove it.